plugin/ThirdParty/OverlayPlugin/dummy.txt

```diff
@@ -1,16 +1,13 @@
-This directory is for DLLs from ngld's OverlayPlugin.
+Please download the latest Zip file from https://github.com/ngld/OverlayPlugin/releases/download/
+Extract the "libs" folder as well as the OverlayPlugin.dll
+
+OverlayPlugin
+|- libs
+|  |- HtmlRenderer.dll
+|  |- Newtonsoft.Json.dll
+|  |- OverlayPlugin.Common.dll
+|  |- OverlayPlugin.Core.dll
+|  |- OverlayPlugin.dll
```

Comment:
`OverlayPlugin.dll` should be
`OverlayPlugin.Updater.dll`

Reply:
I fixed this and added a comment that the tree might be changed by future updates of the dependency.